docker/periodic/daily/zipkin-dependencies-yesterday

```diff
@@ -13,4 +13,4 @@
 # the License.
 #
 
-java ${JAVA_OPTS} -jar /zipkin-dependencies/zipkin-dependencies.jar `date -ud yesterday +%F`
+java ${JAVA_OPTS} -Djava.io.tmpdir=/tmp -cp classes zipkin2.dependencies.ZipkinDependenciesJob `date -ud yesterday +%F`
```

Comment:
Could we re-use the script used by the entry point so we don't have duplication of the command and could hopefully avoid this kind of issue of things getting out of sync?

https://github.com/openzipkin/zipkin-dependencies/blob/50f4c034cb6c8ff86257bfbf229f6129d3d9944c/docker/Dockerfile#L79

It seems we can pass arguments to the `start-zipkin-dependencies` script:
https://github.com/openzipkin/zipkin-dependencies/blob/50f4c034cb6c8ff86257bfbf229f6129d3d9944c/docker/bin/start-zipkin-dependencies#L20

Reply:
That makes much more sense. Adding changes.